src/utils/getImageType.ts

```diff
@@ -0,0 +1,7 @@
+const getImageType = (item: String): String => {
+  const splitedItem = item.split('.');
+
+  return splitedItem[splitedItem.length - 1];
```

Comment:
```suggestion
  return splitedItem.pop();
```
this solution works even if `item.split()` don't actually give more than one element on array

Reply:
Done